libraries/libfc/include/fc/string.hpp

```diff
@@ -2,151 +2,29 @@
 #include <fc/utility.hpp>
 #include <fc/fwd.hpp>
 
-#ifndef USE_FC_STRING
 #include <optional>
 #include <string>
+
 namespace fc
 {
-    typedef std::string string;
-
-  int64_t  to_int64( const fc::string& );
-  uint64_t to_uint64( const fc::string& );
-  double   to_double( const fc::string& );
-  fc::string to_string( double );
-  fc::string to_string( uint64_t );
-  fc::string to_string( int64_t );
-  fc::string to_string( uint16_t );
+  int64_t  to_int64( const std::string& );
+  uint64_t to_uint64( const std::string& );
+  double   to_double( const std::string& );
+  std::string to_string( double );
+  std::string to_string( uint64_t );
+  std::string to_string( int64_t );
+  std::string to_string( uint16_t );
```

Comment:
Why do we still need this string.hpp file? Can we use STD's `to_string`?

Reply:
`to_string( double )` has a "unique" impl that we do rely on it looks like.
The other `to_string` could likely be `std::_to_string` but I would need to verify none of them are used in consensus code before we moved from `lexical_cast` to `std::to_string`.

Reply:
It does look like there are a few more unused functions in there I can kill.